### PR TITLE
Convert Last Tests to Use SceApiTester

### PR DESCRIPTION
**Pull request by law-nguyen**
Refactored test/LedSign.js with new SceApiTester class.

### CODE REVIEW

File: test/LedSign.js [hunk 1]
```diff
@@ -107,87 +107,50 @@ describe('LedSign', () => {
         .to.equal(VALID_SIGN_REQUEST.backgroundColor);
       expect(signResponse.textColor).to.equal(VALID_SIGN_REQUEST.textColor);
       expect(signResponse.borderColor).to.equal(VALID_SIGN_REQUEST.borderColor);
-      done();
     });
-    it('Should return statusCode 404 when the sign is down', done => {
+    it('Should return statusCode 404 when the sign is down', async () => {
       healthCheckMock.rejects(ERROR_MESSAGE);
-      chai
```
Comment (hsorathia) [inline test/LedSign.js:114]: just curious, is the point of this refactor to get rid of chai and implement our own testing scheme

File: test/LedSign.js [hunk 2]
```diff
@@ -107,87 +107,50 @@ describe('LedSign', () => {
...
-        })
-        .catch(err => {
-          throw err;
-        });
+      const response = await test.sendPostRequest(
+        '/api/LedSign/updateSignText', VALID_SIGN_REQUEST);
+      expect(response).to.have.status(OK);
     });
-    it('Should return statusCode 400 when the sign is down', done => {
+    it('Should return statusCode 400 when the sign is down', async () => {
```
Comment (hsorathia) [inline test/LedSign.js:127]: also another dumb question but why do we replace done w/ async

File: test/LedSign.js (hunk 1 above)
Comment (law-nguyen) [reply test/LedSign.js:114]: the refactor decreases the number of lines used for each test. We still call chai, just in another class

File: test/LedSign.js (hunk 2 above)
Comment (law-nguyen) [reply test/LedSign.js:127]: async because the SceApiTester functions are async

Comment (hsorathia) [reply test/LedSign.js:127]: 🤡 ok that makes sense

File: test/LedSign.js (hunk 1 above)
Comment (hsorathia) [reply test/LedSign.js:114]: ok gotchu that makes sense

**Pull request merged**